### DISCUSSION TITLE
Monitoring Costs and Traces generated by RAGAS

### DISCUSSION THREAD

**Discussion by paolotamag** [langfuse/langfuse#2226] [Ideas]
### Describe the feature or potential improvement

hi there,
nice: https://langfuse.com/guides/cookbook/evaluation_of_rag_with_ragas
however it seems to me that the traces created by RAGAS itself (to generate those scores) are not going to langfuse.
I would like to monitor the costs and inspect the traces behind those scores! 
while it's great I can send RAGAS scores to langfuse, can't we monitor RAGAS scoring process itself?

**Comment by marcklingen** [comment]
agree! this'd be cool. we planned to do a deeper collab with the RAGAS team and we also discussed adding support for cost tracking of evals

**Comment by vd-omermazig** [comment]
Any progress with that? Just encountered this too.
